sources/packages/web/src/views/aest/student/StudentNotes.vue

```diff
@@ -10,6 +10,7 @@
           v-model="toggleNotes"
           mandatory
           class="float-right btn-toggle"
+          selected-class="selected-btn-toggle"
```

Comment:
for a header in line 7. can we use ` <header-navigator ` or I am missing anything